packages/vats/src/proposals/probeZcfBundle.js

```diff
@@ -0,0 +1,60 @@
+import { E } from '@endo/far';
+import { getIncarnation } from '@agoric/synthetic-chain';
```

Comment:
I really don't believe this dep can or should be bundled

Reply:
Will you be less unhappy if I copy `getIncarnation`? Will this work in a proposal?
```
export const getIncarnation = async vatName => {
  const fullPath = swingstorePath.replace(/^~/, NonNullish(HOME));
  const kStore = makeSwingstore(dbOpenAmbient(fullPath, { readonly: true }));

  const vatID = kStore.findVat(vatName);
  const vatInfo = kStore.lookupVat(vatID);

  const source = vatInfo.source();
  const { incarnation } = vatInfo.currentSpan();

  // misc info to stderr
  console.error(JSON.stringify({ vatName, vatID, incarnation, ...source }));

  return incarnation;
};
```

Reply:
I mean I don't believe this will work. The proposal executes on chain, you cannot read the state directly from there. Or maybe I'm completely confused.